src/LinuxTracing/ThreadStateManager.cpp

```diff
@@ -40,24 +43,34 @@ void ThreadStateManager::OnNewTask(uint64_t timestamp_ns, pid_t tid) {
     ORBIT_ERROR("Processed task:task_newtask but thread %d was already known", tid);
     return;
   }
-  tid_open_states_.insert_or_assign(tid, OpenState{kNewState, timestamp_ns});
+  tid_open_states_.insert_or_assign(
+      tid, OpenState{kNewState, timestamp_ns, was_created_by_tid, was_created_by_pid,
+                     orbit_grpc_protos::ThreadStateSlice::WakeupReason::
+                         ThreadStateSlice_WakeupReason_kCreated});
```

Comment:
`orbit_grpc_protos::ThreadStateSlice::WakeupReason::kCreated` or `orbit_grpc_protos::ThreadStateSlice::kCreated` (not sure which one works)